sass/chosen.scss

```diff
@@ -147,185 +137,145 @@ $chosen-sprite-retina: image-url('chosen-sprite@2x.png') !default;
 /* @end */
 
 /* @group Results */
-.chosen-container .chosen-results {
-  color: #444;
-  position: relative;
-  overflow-x: hidden;
-  overflow-y: auto;
-  margin: 0 4px 4px 0;
-  padding: 0 0 0 4px;
-  max-height: 240px;
-  -webkit-overflow-scrolling: touch;
-  li {
-    display: none;
+.chosen-container {
+  ul.chosen-results {
+    color: $chosen-text;
+    max-height: 250px;
+    overflow-x: hidden;
+    overflow-y: auto;
     margin: 0;
-    padding: 5px 6px;
-    list-style: none;
-    line-height: 15px;
-    word-wrap: break-word;
-    -webkit-touch-callout: none;
-    &.active-result {
-      display: list-item;
-      cursor: pointer;
-    }
-    &.disabled-result {
-      display: list-item;
-      color: #ccc;
-      cursor: default;
-    }
-    &.highlighted {
-      background-color: #3875d7;
-      @include background-image(linear-gradient(#3875d7 20%, #2a62bc 90%));
-      color: #fff;
-    }
-    &.no-results {
-      color: #777;
-      display: list-item;
-      background: #f4f4f4;
-    }
-    &.group-result {
-      display: list-item;
-      font-weight: bold;
-      cursor: default;
-    }
-    &.group-option {
-      padding-left: 15px;
-    }
-    em {
-      font-style: normal;
-      text-decoration: underline;
+    padding: 0;
+    position: relative;
+    -webkit-overflow-scrolling: touch;
+    li {
+      border-radius: $chosen-radius - 1px;
+      display: none;
+      line-height: 15px;
+      list-style: none;
+      margin: 0;
+      padding: 5px 8px;
+      word-wrap: break-word;
+      -webkit-touch-callout: none;
+      &.active-result {
+        display: list-item;
+        cursor: pointer;
+      }
+      &.disabled-result {
+        display: list-item;
+        color: #ccc;
+        cursor: default;
+      }
+      &.result-selected {
+        display: list-item;
+        &:after {
+          content: " ✔";
```

Comment:
If the character is not supported, it will end up looking like a square. I'm fairly confident that this should be fine for all supported browsers for most people, but you never know how someone might have customized their browser's default fonts. I'm sure there would be a small % of people that this would break for.

To circumvent any frustrations in the future, I'll just update this to use an image, and add a checkmark to both sprites.